### ISSUE TITLE
Seems User Agent Switcher is now unmaintained

### ISSUE THREAD

**Issue by jasan-s** [skratchdot/random-useragent#12]

**Comment by skratchdot** [comment]
thanks for the heads up. i pull the data for this lib from:
http://techpatterns.com/downloads/firefox/useragentswitcher.xml

so i could see that url not working at some point. might need to find a new data source.

**Comment by yidongw** [comment]
yeah looks like the source data hasn't been updated for a long time...
